src/main/resources/site/parts/omStatistikken/omStatistikken.ts

```diff
@@ -0,0 +1,239 @@
+  const background: Category | undefined = content.background
+  const production: Category | undefined = content.production
+  const accuracyAndReliability: Category | undefined = content.accuracyAndReliability
+  const relevantDocumentation: string | undefined = content.relevantDocumentation
+  const aboutSeasonalAdjustment: Category | undefined = content.aboutSeasonalAdjustment
+
+  if (administrativeInformation) {
+    administrativeInformation.nextUpdate = nextUpdate
+  }
+
+  definition && isNotEmpty(definition) ? accordions.push(
+    getAccordion('om-statistikken-definisjoner', phrases.definitions, definition, items.definition, phrases)) : undefined
+  administrativeInformation && isNotEmpty(administrativeInformation) ? accordions.push(
+    getAccordion('om-statistikken-administrative_opplysninger', phrases.administrativeInformation,
+      administrativeInformation, items.administrativeInformation, phrases)) : undefined
+  background && isNotEmpty(background) ? accordions.push(
+    getAccordion('om-statistikken-bakgrunn', phrases.background, background, items.background, phrases)) : undefined
+  production && isNotEmpty(production) ? accordions.push(
+    getAccordion('om-statistikken-produksjon', phrases.production, production, items.production, phrases)) : undefined
+  accuracyAndReliability && isNotEmpty(accuracyAndReliability) ? accordions.push(
+    getAccordion('om-statistikken-feilkilder', phrases.accuracyAndReliability,
+      accuracyAndReliability, items.accuracyAndReliability, phrases)) : undefined
+
+  const relevantDocumentationAccordion: Accordion = {
+    id: 'om-statistikken-relevant-dokumentasjon',
+    body: relevantDocumentation ? processHtml({
+      value: relevantDocumentation.replace(/&nbsp;/g, ' ')
+    }) : undefined,
+    open: phrases.relevantDocumentation,
+    items: []
+  }
+
+  if (relevantDocumentation) {
+    accordions.push(relevantDocumentationAccordion)
+  }
+
+  aboutSeasonalAdjustment && isNotEmpty(aboutSeasonalAdjustment) ? accordions.push(
+    getAccordion('om-sesongjustering', phrases.aboutSeasonalAdjustment, aboutSeasonalAdjustment,
+      items.aboutSeasonalAdjustment, phrases)) : undefined
+
+  return accordions
+}
+
+function getAccordion(id: string, categoryText: string, category: Category, items: Array<string>, phrases: Phrases): Accordion {
+  return {
+    id: id,
+    body: '',
+    open: categoryText,
+    items: getItems(category, items, phrases)
+  }
+}
+
+function getItems(category: Category, variables: Array<string>, phrases: Phrases): Array<AccordionItem> {
+  return variables.map((variable) => {
+    return {
+      title: phrases[variable],
+      body: category[variable] ? processHtml({
+        value: category[variable].replace(/&nbsp;/g, ' ')
+      }) : phrases.notRelevant
+    }
+  })
+}
+
+function isNotEmpty(obj: object | undefined): boolean {
+  if (obj) {
+    return Object.keys(obj).length > 0
+  }
+  return false
+}
+
+interface Accordion {
```

Comment:
Suggestion: Could the Accordion interfaces be moved to /lib/types/components file so that we can use them for other/future parts?